manifests/crc/BYODB-installation/minikube/kruize-crc-minikube.yaml

```diff
@@ -25,6 +45,9 @@ spec:
         - name: kruize
           image: kruize/autotune_operator:0.0.9_rm
```

Comment:
Should we update the image tag to 0.0.10_rm?

Reply:
@chandrams as per the process we have to run tests on 0.0.10_mvp and if the tests are successful, tag and push 0.0.10_rm and then change this to 0.0.10_rm. @msvinaykumar I'd suggest you revert it back. Sorry about that

Reply:
Thanks Dino for pointing this out. @msvinaykumar - Please revert it back